com.unity.render-pipelines.universal/Runtime/2D/LightUtility.cs

```diff
@@ -170,127 +171,250 @@ public static Bounds GenerateSpriteMesh(Mesh mesh, Sprite sprite)
             mesh.SetVertexBufferParams(vertices.Length, SpriteLightMeshVertex.VertexLayout);
             mesh.SetVertexBufferData(vertices, 0, 0, vertices.Length);
             mesh.SetIndices(srcIndices, MeshTopology.Triangles, 0, true);
-            return mesh.GetSubMesh(0).bounds;
+            return mesh.bounds;
```

Comment:
Are you sure about this change? According to my previous testing, the last boolean parameter of `Mesh.SetIndices()` only tells the mesh to recalculate the bounds for the submesh in question, not for the whole mesh. Returning Mesh.bounds here was causing culling issues in Lost Crypt.

Reply:
I think you are right. Let me check and reset it back if its the case.

Reply:
mesh.bounds encapsulates bounds of all submeshes. We can also get bounds for each Submesh too. For now I have changed as you suggested.

I guess once we have Transparent/Opaque Split we may need to use mesh.bounds.